addon/helpers/filter.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import { filter } from 'ember-computed';
+import Helper from 'ember-helper';
+import get from 'ember-metal/get';
+import observer from 'ember-metal/observer';
+import set from 'ember-metal/set';
+import { isEmpty } from 'ember-utils';
+
+const { defineProperty } = Ember;
+
+export default Helper.extend({
+  compute([callback, array]) {
+
```

Comment:
@poteto sorry, I'm not sure what you mean. I checked for extra white space and I don't see it. I also looked at how `filter-by` is implemented and I don't see a difference there either. Could you please tell me what you'd like me to change about the whitespace.